cmd/metadata/main.go

```diff
@@ -212,3 +224,33 @@ func createViews(ctx context.Context, database golibConfig.Database) ([]string,
 
 	return views, nil
 }
+
+func readCustomHasuraConfigs(ctx context.Context, database golibConfig.Database) ([]hasura.Request, error) {
+	files, err := ioutil.ReadDir("custom_hasura_config")
+	if err != nil {
+		return nil, err
+	}
+
+	custom_configs := make([]hasura.Request, 0)
+	for i := range files {
+		if files[i].IsDir() || strings.HasPrefix(files[i].Name(), ".") {
+			continue
+		}
+
+		path := fmt.Sprintf("custom_hasura_config/%s", files[i].Name())
+		raw, err := ioutil.ReadFile(path)
```

Comment:
Reading full file is a bad practice, because you load it to memory. If file is large (for example, 10GB), you will get some troubles. The solution is using streams. For example:
```go
f , err := os.OpenFile(filename)
if err != nil {
   return err
}
defer f.Close()

var data any  // YOUR TYPE HERE
if err := json.NewDecoder(f).Decode(&data); err != nil {
   return err
}
```

It's important, because you provide opportunity to add files for users and you don't know size of their files.

Reply:
Again, was going by how sql views are done. Hasura config files should not be gigabytes in size (more along the lines of sql views in size).